tests/MyEventEmitter.test.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const MyEventEmitter = require('../src/MyEventEmitter');
+const EventEmitter = require('events');
+
+describe('MyEventEmitter', () => {
+  let emitter;
+
+  beforeEach(() => {
+    emitter = new MyEventEmitter();
+  });
+
+  test('should not use external libraries', () => {
+    const sourceFile = path
+      .resolve(__dirname, '..', 'src', 'MyEventEmitter.js');
+    const fileContent = fs.readFileSync(sourceFile, 'utf8');
+    const hasCommonJSRequire = /require\(/g.test(fileContent);
+    const hasESImports = /import\s+\S+\s+from\s+['"].+['"]/g.test(fileContent);
+
+    expect(hasCommonJSRequire).toBe(false);
+    expect(hasESImports).toBe(false);
+  });
+
+  test('should not use Node.js EventEmitter', () => {
+    expect(emitter).not.toBeInstanceOf(EventEmitter);
+  });
+
+  test('should have working "listenerCount" method', () => {
```

Comment:
should have XXX method is not a good test name
because it does not tell what functionality of the method is tested